src/Digdir.Domain.Dialogporten.Infrastructure/Persistence/Configurations/Actors/PopulateActorNameInterceptor.cs

```diff
@@ -0,0 +1,79 @@
+using Digdir.Domain.Dialogporten.Application.Common;
+using Digdir.Domain.Dialogporten.Application.Externals;
+using Digdir.Domain.Dialogporten.Domain.Actors;
+using Digdir.Domain.Dialogporten.Domain.Dialogs.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Diagnostics;
+
+namespace Digdir.Domain.Dialogporten.Infrastructure.DomainEvents.Outbox;
+
+internal sealed class PopulateActorNameInterceptor : SaveChangesInterceptor
+{
+    private static readonly Type ActorType = typeof(Actor);
+
+    private readonly IDomainContext _domainContext;
+    private readonly IPartyNameRegistry _partyNameRegistry;
+
+    public PopulateActorNameInterceptor(
+        IDomainContext domainContext,
+        IPartyNameRegistry partyNameRegistry)
+    {
+        _domainContext = domainContext ?? throw new ArgumentNullException(nameof(domainContext));
+        _partyNameRegistry = partyNameRegistry ?? throw new ArgumentNullException(nameof(partyNameRegistry));
+    }
+
+    public override async ValueTask<InterceptionResult<int>> SavingChangesAsync(
+        DbContextEventData eventData,
+        InterceptionResult<int> result,
+        CancellationToken cancellationToken = default)
+    {
+        var dbContext = eventData.Context;
+
+        if (dbContext is null)
+        {
+            return await base.SavingChangesAsync(eventData, result, cancellationToken);
+        }
+
+        var actors = dbContext.ChangeTracker.Entries()
+            .Where(x => x.Metadata.ClrType.IsAssignableTo(ActorType))
+            .Where(x => x.State is EntityState.Added or EntityState.Modified)
+            .Select(x =>
+            {
+                var actor = (Actor)x.Entity;
+                actor.ActorId = actor.ActorId?.ToLowerInvariant();
+                return actor;
+            })
+            .Where(x => !string.IsNullOrWhiteSpace(x.ActorId))
```

Comment:
```suggestion
            .Where(x => x.ActorId is not null)
```
Skal vel her ikke kunne være annet enn `null` eller en well-formed id.